lib/integrations/awsoidc/deploydatabaseservice.go

```diff
@@ -259,11 +259,9 @@ func DeployDatabaseService(ctx context.Context, clt DeployServiceClient, req Dep
 		}
 	}
 
-	clusterDashboardURL := fmt.Sprintf("https://%s.console.aws.amazon.com/ecs/v2/clusters/%s/services", req.Region, aws.ToString(cluster.ClusterName))
-
 	return &DeployDatabaseServiceResponse{
 		ClusterARN:          aws.ToString(cluster.ClusterArn),
-		ClusterDashboardURL: clusterDashboardURL,
+		ClusterDashboardURL: ecsClusterDashboardURL(req.Region, aws.ToString(cluster.ClusterName)),
```

Comment:
should we also change this to use `ECSDatabaseServiceDashboardURL`? since we are now limiting to one vpc deployment per flow? 🤔

Reply:
I think that makes sense

Reply:
looking at this, I think it would be appropriate to address this in another PR than this one.
It's a little awkward to use a service specific link when the aws oidc service handler request is a list of deployments.

So I'll skip this for now.

Reply:
future reference: added this suggestion in https://github.com/gravitational/teleport/pull/44707